packages/website-frontend/src/app/repository/repository.component.html

```diff
@@ -1 +1,48 @@
-<div>{{ value.slug }}</div>
\ No newline at end of file
+<div class="row">
+  <div class="col-md-10">{{ repo.slug }}</div>
+  <div class="col-md-2">
+    <input #checkbox type="checkbox" class="toggle-switch hidden" [checked]="repo.enabled" />
```

Comment:
Do you mean the `#checkbox` attribute? It can be removed. I used to use it to do something programmatically with the checkbox, but that's no longer necessary.